website/docs/docs/collaborate/govern/project-dependencies.md

```diff
@@ -7,24 +7,25 @@ pagination_next: null
-- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). You need at least one successful job run after defining their `access`.
-- Define a deployment environment in the upstream ("producer") project [that is set to be your Production environment](/docs/deploy/deploy-environments#set-as-production-environment), and ensure it has at least one successful job run in that environment.
-- If the upstream project has a Staging environment, run a job in that Staging environment to ensure the downstream cross-project ref resolves.
-- Each project `name` must be unique in your dbt Cloud account. For example, if you have a dbt project (codebase) for the `jaffle_marketing` team, you should not create separate projects for `Jaffle Marketing - Dev` and `Jaffle Marketing - Prod`. That isolation should instead be handled at the environment level.
-  - We are adding support for environment-level permissions and data warehouse connections; please contact your dbt Labs account team for beta access.
-- The `dbt_project.yml` file is case-sensitive, which means the project name must exactly match the name in your `dependencies.yml`.  For example, if your project name is `jaffle_marketing`, you should use `jaffle_marketing` (not `JAFFLE_MARKETING`) in all related files.
-
+- Available in [dbt Cloud Enterprise](https://www.getdbt.com/pricing). To use it, designate a [public model](/docs/collaborate/govern/model-access) and add a [cross-project ref](#how-to-write-cross-project-ref).
+- Upstream ("producer") project setup:
+  - Configure models in upstream project with [`access: public`](/reference/resource-configs/access) and have at least one successful job run after defining `access`.
+  - Define a deployment environment in the upstream project as [Production environment](/docs/deploy/deploy-environments#set-as-production-environment), ensuring at least one successful job run in that environment.
+  - If the upstream project has a Staging environment, run a job in that Staging environment to ensure the downstream cross-project ref resolves.
+- Each project `name` must be unique in your dbt Cloud account. For example, if you have a dbt project (codebase) for the `jaffle_marketing` team, avoid creating projects for `Jaffle Marketing - Dev` and `Jaffle Marketing - Prod`; use [environment-level isolation](/docs/dbt-cloud-environments#types-of-environments) instead.
+  - dbt Cloud supports [Global connections](/docs/cloud/connect-data-platform/about-connections#connection-management), allowing different data platform connections per environment for a single project. This eliminates the need to duplicate projects.
```

Comment:
@ericbaumann -- is the feature name 'global connections'? i don't see it mentioned in the cloud ui and docs either so wanting to confirm to make any corrections 🙏

Reply:
I think we can just call them connections.

A couple caveats Eric can confirm... you can only use multiple connections within a project of the same warehouse type (all snowflake, all databricks, etc). Additionally, the connections are reusable across projects/environments which reduces the amount you need to create.

I think the second point is really where the term global comes from, but that's more internal naming than public facing. I reframe the callout to not include global but maybe say reusable connection definitions or something that is more clear.

Reply:
I would agree that global is kind of an internal term. If you're looking for something public facing, I would say "account level connections" or just "connections"

I'm on my phone at the moment but the other thing I'd mention is that I don't believe this feature is enterprise only. it's the new way for doing connections across all plan tiers

Reply:
this blurb feels like it's mostly about mesh setup. 

It's probably good to mention this there, but is there a more generalized place we talk about connections that should also be updated?

Reply:
thanks so much @thorn14 and @ericbaumann ! 

Eric, for your comments -- i can clarify that connections is available for all but also think its worth mentioning it in the mesh prerequisites. 

in terms of other places to update -- we have the main doc about [connection management](https://docs.getdbt.com/docs/cloud/connect-data-platform/about-connections#connection-management) and it looks like the below doc might need that?

- https://docs.getdbt.com/docs/deploy/deploy-environments#deployment-connectionnnections#connection-management

![Screenshot 2025-03-07 at 14 35 32](https://github.com/user-attachments/assets/828c2687-e6dc-4e0e-baea-7008ed7aad33)

Reply:
thank you @thorn14 ! going to update it to the below if you both think that makes sense:

> dbt Cloud supports [Connections](/docs/cloud/connect-data-platform/about-connections#connection-management), available to all dbt Cloud users. Connections allows different data platform connections per environment, eliminating the need to duplicate projects. Projects can only use multiple connections of the same warehouse type. Connections are reusable across projects and environments.

Reply:
Yeah i think modifying that `Connection Management` page to say connections are created and managed at the account level, but are then assigned to an environment. Currently, we recommend that you create a new environment if you are looking to change warehouse type. 

This allows you to use _more_ than one connection per project like it says today.

Reply:
great thanks @ericbaumann ! so is teh below a good replacement for the callout:

> Warehouse connections are created and managed at the account-level for dbt Cloud accounts and assigned to an environment. To change warehouse type, we recommend creating a new environment.  
 Each Project can have multiple connections (Snowflake account, Redshift host, Bigquery project, Databricks host, and so on.) of the same warehouse type. Some details of that connection (databases/schemas/etc.) can be overridden within this section of the dbt Cloud environment settings.

Reply:
That looks good, thanks @mirnawong1!